docker/pravega/scripts/init_kubernetes.sh

```diff
@@ -45,29 +45,34 @@ init_kubernetes() {
 
         local ns=${POD_NAMESPACE}
         local podname=${POD_NAME}
+        if [ -z "${SERVICE_NAME}" ]; then
```

Comment:
Could we add a comment indicating that the SERVICE_NAME is set by the pravega operator when external access is enabled?.

Reply:
Added comment indicating same